src/main/webapp/constants/errorCodes.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Rename this file back to `.js` as we don't need to use jsx syntax here.

Reply:
<img width="1022" alt="Screenshot 2021-06-08 at 11 08 58 PM" src="https://user-images.githubusercontent.com/69460835/121232139-e4988e80-c8ae-11eb-842d-fd51c6f4d89b.png">

It works but will throw a syntax error for obvious reasons.

Reply:
My bad. I thought we only kept strings.